### ISSUE TITLE
"Disabled incompatible app" and "Error updating .htaccess file"

### ISSUE THREAD

**Issue by luisBSBDATA** [nextcloud/vm#654]
## Steps to reproduce
1. Set up an Ubuntu Server 18.04
2. Install Nextcloud VM using the provided script (nextcloud_install_production.sh)
3. Choose to install "Calendar", "Contacts", "CanIUpdate", "IssueTemplate" and "Passman" as additional apps

## Expected behaviour
Finish the installation without errors

## Actual behaviour
It shows those errors:
![image](https://user-images.githubusercontent.com/37793591/45148741-41c7f880-b1c8-11e8-8470-94f2a9b2d725.png)
![image](https://user-images.githubusercontent.com/37793591/45148763-4d1b2400-b1c8-11e8-9325-76cb0dad2b4c.png)

## Server configuration

**Server version:** Ubuntu Server 18.04

**How did you install the VM?** From the script

**Comment by enoch85** [comment]
Disabling incomaptible apps [is not a bug, it's a feature](http://poster.keepcalmandposters.com/9013580.jpg).

I will check the other error.

**Comment by enoch85** [comment]
Nope, everything works here: https://0bin.net/paste/pnWR-AUwXs5NdMit#zFLjwi2nwZS0JK3orexadrKpYiIe-ce69y9ok9rsvCB

**Comment by nxtester** [comment]
I can confirm the following error where script will break:

![teamviewer_2018-09-06_21-35-56](https://user-images.githubusercontent.com/28905767/45180852-f7bd3200-b21c-11e8-959d-b55171137d62.png)

**Comment by enoch85** [comment]
@luisBSBDATA Ok, so is that when doing an upgrade from 13.0.6 on Ubuntu 16.04?

Please give me the exact steps to reproduce.

**Comment by enoch85** [comment]
How does your `overwrite.cli.url` look like?

**Comment by nxtester** [comment]
Nope, clean install from clean reference plattform "Ubuntu Server 18.04 LTS 64-bit " according to recommended specs:

    Thick provisioned (better performance and easier to maintain)
    DHCP available
    40 GB HDD for OS
    4 vCPU
    4 GB RAM

I use this reference VM for some weeks and had never a platform problem.

**Comment by enoch85** [comment]
@nxtester That's very strange. I installed 13.0.6 yesterday using the scripts, and today I updated that VM. 

So, how can we do the exact same thing and get different results?

Is this during updating or installing?

**Comment by nxtester** [comment]
During second part after first reboot.

**Comment by enoch85** [comment]
> During second part after first reboot.

Please be more specific. Do you mean in the startup script?

**Comment by nxtester** [comment]
I the last 3 days I tried every day a new installation w/o success. Errors have been communicated and approved. Sorry, I am not a linux guy. OK, I started "sudo bash nextcloud_install_production.sh" and it went fine until first reboot. Then I logged in and second part started, first explanation about all steps (big information box) and then after a while script stopped by error.

**Comment by nxtester** [comment]
Can do it again, made a snapshot after successful first stage / reboot.

**Comment by nxtester** [comment]
First initial user (standard in reference VM) was "linux", second user made by script was "ncadmin". Script continued then with "ncadmin". Maybe important to know?

**Comment by nxtester** [comment]
This is what I call "stage 2 after first reboot":

![putty_2018-09-06_21-55-20](https://user-images.githubusercontent.com/28905767/45181759-a7939f00-b21f-11e8-99e2-779284a8da51.png)

**Comment by enoch85** [comment]
I found the error and will fix it.

Thanks for reporting!

**Comment by enoch85** [comment]
@nxtester Can you please path your setup script with the fix in the PR and see it it works?

**Comment by luisBSBDATA** [comment]
> Disabling incompatible apps is not a bug, it's a feature.

Oh, okay. So I assume that I'll be able to update and activate them from Nextcloud in the future, right?

> @nxtester Can you please path your setup script with the fix in the PR and see it it works?

It's already fixed then? I will rollback my VM to a clean state and try again, then I will get back to you. Thanks again!

**Comment by enoch85** [comment]
> It's already fixed then? I will rollback my VM to a clean state and try again, then I will get back to you. Thanks again!

Please do!

**Comment by nxtester** [comment]
Checked it and changed script. Error seemes to be fixed, but anyway not possible to do finish regularly. Here is the next error:

![putty_2018-09-07_10-05-44](https://user-images.githubusercontent.com/28905767/45206636-caf13500-b285-11e8-897c-e753121a7e9a.png)

Thanks for your appreciated work!

**Comment by luisBSBDATA** [comment]
Tried again and this is what I got:

![image](https://user-images.githubusercontent.com/37793591/45211662-2970e000-b293-11e8-82c4-65117f7d14fc.png)

**Comment by enoch85** [comment]
@luisBSBDATA OK, can you please give me the exact steps to reproduce? The fix worked for me.

**Comment by luisBSBDATA** [comment]
> @luisBSBDATA OK, can you please give me the exact steps to reproduce? The fix worked for me.

- I rolled back my VM to a clean state.

- I re-download the "nextcloud_install_production.sh" script and executed it

- I ran through all the install process until the first reboot, then it failed just after asking me if I wanted to change the timezone. The other error was further than that new one.

**Comment by luisBSBDATA** [comment]
Okay, I thought it was already merged with master. Let me try again then :)

**Comment by luisBSBDATA** [comment]
Now that's what I got if I choose to install Passman:

![image](https://user-images.githubusercontent.com/37793591/45223216-f50f1b00-b2b6-11e8-85f4-6411e7cda21e.png)

**Comment by nxtester** [comment]
Same here.

**Comment by enoch85** [comment]
Great! So the .htaccess issue is gone?

For the other error: https://github.com/nextcloud/server/issues/11070

**Comment by nxtester** [comment]
> So the .htaccess issue is gone?

For me I can approve that .htaccess issue was gone after modifying script.

**Comment by nxtester** [comment]
BTW one question: Is there any disadvantage to uncheck all NC apps (lika Passman) and install it later by NC portal?

**Comment by luisBSBDATA** [comment]
> Great! So the .htaccess issue is gone?

Yeah, thanks one more time @enoch85 for your time and patience

> BTW one question: Is there any disadvantage to uncheck all NC apps (lika Passman) and install it later by NC portal?

Im interested on it too

**Comment by enoch85** [comment]
> BTW one question: Is there any disadvantage to uncheck all NC apps (lika Passman) and install it later by NC portal?

What the script does is installing the app from App Store via CLI (the Nextcloud built in `occ` command), so no, there are no difference.

Will release Nextcloud 14 soon!

**Comment by enoch85** [comment]
And guys, THANKS for testing!

**Comment by nxtester** [comment]
Thanks to you, Enoch85. We are only helping a little. 

> Will release Nextcloud 14 soon!

Soon or sooner? ;-))

**Comment by luisBSBDATA** [comment]
Nice, I will install them manually post-installation then. Thank YOU @enoch85, I will keep on posting if I find out something else 👍
